script/foreman-debug

```diff
@@ -79,6 +83,21 @@ add_cmd() {
   eval $CMD >> "$DIR/$OUT" 2>&1
 }
 
+# add outout of the sql and redirect possible errors there
+add_sql() {
+  SQL=$1
+  OUT=$2
+  printv " - $OUT"
+  echo -e "SQL> $SQL\n" > "$DIR/db_$OUT"
+  if [ "$DB_ADAPTER" = "postgresql" ]; then
```

Comment:
Database dump is a good idea, but I will implement that separately for the reasons above.